Tools/pyco/pyco.py

```diff
@@ -362,10 +355,25 @@ def preload(self):
             consts = code.co_consts[1:]
         else:
             consts = code.co_consts
-        for value in consts:
+
+        self.co_consts_start = len(self.builder.constants)
+        self.builder.co_consts_start = self.co_consts_start
+        for i, value in enumerate(consts):
             if not is_immediate(value):
-                self.builder.add_constant(value)
+                index = self.builder.add_constant(value)
+            else:
+                # not used, but keeps it simple with the indices
```

Comment:
I don't think what I'm doing here for immediates to correct. It will confuse get_bytes(). We probably need to put some "no-value" thing in the constants array.